overlay_gl/init_unix.c

```diff
@@ -6,6 +6,16 @@
 // This file is included by overlay.c for
 // Unix/X11/GLX-specific overlay initialization.
 
+#include <inttypes.h>
+
+#if defined(__linux__)
+// ELF32_ST_TYPE and ELF64_ST_TYPE are the same.
```

Comment:
Is this always the case or maybe just a coincidence on your system? If they are the same, why should there by 2 types in the first place?

Reply:
It's always the case, it's how the ELF header is. In fact, the current code only uses `ELF32_ST_TYPE`:
https://github.com/mumble-voip/mumble/blob/caaf516ea50096e10476e76ad20ae2ef8505304b/overlay_gl/init_unix.c#L186-L187

From `/usr/include/elf.h`:
```c
/* How to extract and insert information held in the st_info field.  */

#define ELF32_ST_BIND(val)		(((unsigned char) (val)) >> 4)
#define ELF32_ST_TYPE(val)		((val) & 0xf)
#define ELF32_ST_INFO(bind, type)	(((bind) << 4) + ((type) & 0xf))

/* Both Elf32_Sym and Elf64_Sym use the same one-byte st_info field.  */
#define ELF64_ST_BIND(val)		ELF32_ST_BIND (val)
#define ELF64_ST_TYPE(val)		ELF32_ST_TYPE (val)
#define ELF64_ST_INFO(bind, type)	ELF32_ST_INFO ((bind), (type))
```